Sources/DataPipeline/DataPipelineImplementation.swift

```diff
@@ -252,3 +251,88 @@ extension DataPipelineImplementation {
         return attributesPlugin
     }
 }
+
+// To process pending tasks in background queue
+// BGQ in each of the following methods refer to background queue
+extension DataPipelineImplementation {
+    func processIdentifyFromBGQ(identifier: String, timestamp: String, body: [String: Any]?) {
+        var identifyEvent = IdentifyEvent(userId: identifier, traits: nil)
+        identifyEvent.timestamp = timestamp
```

Comment:
just make sure we have verified the format of timestamp that we are adding from BQ is same as one Analytics expect

Reply:
Updated all timestamps to ISO format.